bigtable/CMakeLists.txt

```diff
@@ -47,6 +55,19 @@ include(${PROJECT_SOURCE_DIR}/cmake/EnableClangTidy.cmake)
 # We use abseil.io .
 set(BUILD_TESTING OFF)
 include(${PROJECT_SOURCE_DIR}/cmake/IncludeAbseil.cmake)
+# config options
+if (WIN32)
+  # /wd4005  macro-redefinition
+  # /wd4068  unknown pragma
+  # /wd4244  conversion from 'type1' to 'type2'
+  # /wd4267  conversion from 'size_t' to 'type2'
+  # /wd4800  force value to bool 'true' or 'false' (performance warning)
+  add_compile_options(/W3 /wd4005 /wd4068 /wd4244 /wd4267 /wd4800)
+  # This to avoid conflict with 'max' as it has been made a macro for windows
+  add_definitions(/DNOMINMAX /DWIN32_LEAN_AND_MEAN=1 /D_CRT_SECURE_NO_WARNINGS)
+else()
+  set(ABSL_STD_CXX_FLAG "-std=c++11" CACHE STRING "c++ std flag (default: c++11)")
```

Comment:
It seems like this is motivated by Abseil?  Can you move that to the IncludeAbseil.cmake file if that is the case?

Reply:
Done.